frontend/integration-tests/protractor.conf.ts

```diff
@@ -120,6 +120,7 @@ const testSuites = {
     'tests/crd-extensions.scenario.ts',
   ]),
   clusterSettings: suite(['tests/cluster-settings.scenario.ts']),
+  alertmanager: suite(['tests/alertmanager.scenario.ts']),
```

Comment:
Are we sure we want such detail tests to run for `e2e`?  Not sure testing how global values are saved should be part of CI.   `monitoring` which is part of `e2e` tests the [PagerDuty] form controls.  `alertmanager` test global values for all 4 receiver forms.

Reply:
> Are we sure we want such detail tests to run for e2e?

Why wouldn't we want better test coverage for the merge queue? The only reason I can think of holding them out is because they're not reliable, in which case we should fix the tests.

Reply:
If the tests don't gate merge, I'm be shocked if they didn't break over time. We took time to write the tests. Let's get the value from them.

Reply:
fixed